README.md

```diff
@@ -8,13 +8,13 @@ monitoring system and alerting toolkit.
 
 This repository contains a [Juju](https://jaas.ai/) Charm for
 deploying the monitoring component of Prometheus in a Kubernetes
-cluster. The alerting component of prometheus is offered through a
+cluster. The alerting component of Prometheus is offered through a
 separate Charm.
 
 
 ## Usage
 
-The Prometheus Operator may be deployed using the Juju commandline as
+The Prometheus Operator may be deployed using the Juju command line as
 in
 
     juju deploy prometheus-k8s
```

Comment:
```sh
$ juju deploy prometheus-k8s
```

Reply:
OK. Looks like you are suggesting dollar sign again, to indicate shell prompt.